client/src/components/related/Related.jsx

```diff
@@ -4,19 +4,19 @@ import RelatedProductsList from './RelatedProductsList.jsx';
 import products from '../../sample-data/products.js';
 import axios from 'axios';
 
-const Related = () => {
+const Related = (props) => {
   // *************
   // State
   //**************
   let [relatedProductsId, setProductsId] = useState([]);
-  let [relatedProductsDetail, setDetail] = useState([]);
+  let [relatedProducts, setDetail] = useState([]);
 
   // *************
   // Initial Renders of Data
   // *************
   useEffect(() => {
     axios.get('/related', {params: {
-        product_id: 63609
+        product_id: props.product.id
```

Comment:
In my PR #32, I updated the `product_id` state variable to just be the actual product ID integer.

So you'll want to pull in the changes from the main branch, merge them into your branch, then update this line to be: `product_id: props.product_id`